packages/http/libraries/core/src/http/adapter/InversifyHttpAdapter.ts

```diff
@@ -41,6 +41,17 @@ export abstract class InversifyHttpAdapter<
     await this.#registerControllers();
   }
 
+  #buildLogger(httpAdapterOptions: HttpAdapterOptions | undefined): Logger {
```

Comment:
Maybe, parameter should be optional? like `parseHttpAdapterOptions`. Or `parseHttpAdapterOptions` should have a posible undefined parameter. idk, I'm just talking about consistency

Reply:
My current approach is providing optional parameters to public interfaces, but I honeslty don't know how strict I'm about being consistent in this case :sweat_smile:

Reply:
That's a optimal approach for me. I will refactor in future with that standard in mind